core/test/XiosFile_test.cpp

```diff
@@ -83,6 +83,10 @@ MPI_TEST_CASE("TestXiosFile", 2)
     xios_handler.setFileOutputFreq(fileId, freq);
     REQUIRE(xios_handler.isDefinedFileOutputFreq(fileId));
     REQUIRE(xios_handler.getFileOutputFreq(fileId) == freq);
+    // Split frequency
+    const std::string splitFreq { "1ts" };
```

Comment:
Is this a typical way of setting a std::string? I can see that it works, and how, but I would always use
```
const std::string splitFreq = "1ts";
```
I'd suggest we stick to one or the other in the code base. If the `{}` version is currently preferred, then I'll switch all my `const std::string`s over as I touch their files.

Reply:
Oh good point. In other tests I followed Tom's approach elsewhere, which read like
```
std::string splitFreq = { "1ts"};
```
I'd meant to use the same notation but must have reverted to this instead.

While we're on the topic, though, the C++ course I did recently did recommend initialising variables with
```
std::string splitFreq { "1ts"};
```
rather than
```
std::string splitFreq = "1ts";
```
There are some reasons behind this listed here: https://stackoverflow.com/questions/18222926/what-are-the-advantages-of-list-initialization-using-curly-braces. (TLDR: _Prefer {} initialization over alternatives unless you have a strong reason not to._)

Reply:
Excellent, thank you :)